src/TestImages.jl

```diff
@@ -77,6 +77,10 @@ $(reduce((x, y)->join([x, "\n - \`\"", splitext(y)[1], "\"\`"]), sort(remotefile
 function testimage(filename; download_only::Bool = false, ops...)
     imagefile = image_path(full_imagename(filename))
 
+    if startswith(basename(imagefile), "lena_")
+        @warn "Usage of \"lena\" is not recommended, and the image may be removed in a later release. See https://womenlovetech.com/losing-lena-why-we-need-to-remove-one-image-and-end-techs-original-sin/ for more information." maxlog=1
```

Comment:
Here we use an explicit warning instead of a depwarn because depwarn is by default not showing up after Julia 1.5.